db/migrations/20190527150600_create_slug_events/up.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE events
+    ADD slug VARCHAR(255);
+
+CREATE UNIQUE INDEX index_events_slug ON events (slug);
+
+UPDATE events
+SET slug= substr(lower(replace(name, ' ', '-')), 255);
```

Comment:
This is going to fail if there are any duplicate event names (which we'll have for sure in dev and staging) it is also taking the substr from 249 to end of string
What about something like:
```suggestion
SET slug = CONCAT(SUBSTR(LOWER(REPLACE(name, ' ', '-')), 0, 249), '-', SUBSTR(MD5(RANDOM()::TEXT), 0, 6));
```